radio/src/CMakeLists.txt

```diff
@@ -979,14 +979,15 @@ if(ARCH STREQUAL ARM)
   set(CMAKE_ASM_COMPILE_OBJECT "<CMAKE_ASM_COMPILER> -o <OBJECT> <SOURCE>")
 
   set(COMMON_FLAGS "-mcpu=${MCU} -mthumb -fomit-frame-pointer -fverbose-asm -Wa,-ahlms=firmware.lst -O${OPT} -gdwarf-2 -DHSE_VALUE=${HSE_VALUE} -fno-exceptions -fdata-sections -ffunction-sections ${WARNING_FLAGS}")
+  set(COMMON_CPP_FLAGS "${COMMON_FLAGS} -std=gnu++11")
```

Comment:
Don't forget to add a comment in the CMakeLists.txt otherwise I will forget and change it next time I modify the file!